circuit-lib/circuit-lib.js/package.json

```
@@ -18,6 +19,7 @@
     "ffjavascript": "^0.2.62"
   },
   "devDependencies": {
+    "@coral-xyz/anchor": "0.28.0",
```

```suggestion
    "@coral-xyz/anchor": "0.29.0",
```

We did an upgrade already